### ISSUE TITLE
[Screen Reader-Projects-My Stuff-Import]: Narrator is not providing the information about the Import file dialog box.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2522]
**Pre-Requisite:** Turn on Narrator(Win + Ctrl + Enter)

**User Experience:** Users who depends on Screen Reader will get confuse if Narrator will not provide any information about the dialog box.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool used:-IE Developer (F12)

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to the MY Stuff tab on the page opened and select Import option.
5. Navigate to various controls lying on the page opened..
6. Verify that when Import dialog box get opened Narrator is informing or not.
 
**Actual Result:-**
When Import dialog box get opened Narrator does not provides any information about the dialog box.

**Expected Result:-**
When Import dialog box get opened Narrator should announce the dialog box name like Import file dialog box.
 
**Note:** This same issue does not exists with NVDA + FF

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Please refer the attachment.**

[MAS4.2.1_Projects_Import file.zip](https://github.com/Microsoft/pxt/files/1176057/MAS4.2.1_Projects_Import.file.zip)

**Comment by veler** [comment]
@v-amik Issue fixed and available to test

**Comment by v-amik** [comment]
As per process, bug should be assigned to HCL for regression.

For Any external/Won’t Fix/By Design bug, there is a process to go through by Product team. 
Please go through the process here:

https://microsoft.sharepoint.com/teams/ceacc/SitePages/Exception process.aspx
https://microsoft.sharepoint.com/teams/ceacc/SitePages/External Bug Process.aspx

**Comment by v-mesala** [comment]
Bug is verified on URL :https://makecode.microbit.org/acc bug is fixed hence closed the bug and added the closed tag.
